dockerfiles/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.7'
+
+services:
+  fuseki:
+    container_name: skosmos-fuseki
+    hostname: fuseki
+    image: stain/jena-fuseki
+    environment:
+      - ADMIN_PASSWORD=admin
+      - JVM_ARGS=-Xmx2g
+    ports:
+      - 9030:3030
+  # You can uncomment the line below to have a local volume bound onto the container, or
+  # visit https://hub.docker.com/r/stain/jena-fuseki/ for other alternatives
+  # volumes:
+  #  - ${PWD}/fuseki:/fuseki
+    volumes:
+      - type: bind
+        source: ./config/skosmos.ttl
+        target: /fuseki/configuration/skosmos.ttl
+  fuseki-cache:
+    container_name: skosmos-fuseki-cache
+    hostname: fuseki-cache
+    image: varnish
+    ports:
+      - 9031:80
+    volumes:
+      - type: bind
+        source: ./config/varnish-default.vcl
+        target: /etc/varnish/default.vcl
+  skosmos:
+    container_name: skosmos-web
+    hostname: skosmos
+    build:
+      context: ../
+      dockerfile: dockerfiles/Dockerfile.ubuntu
+    ports:
+      - 9090:80
+    depends_on:
+      - fuseki
+      - fuseki-cache
+    volumes:
+      - type: bind
+        source: ../dockerfiles/config/config-docker-compose.ttl
+        target: /var/www/html/config.ttl
+      - ..:/var/www/html
+
+# once the environment has been created:
+# 1. create dataset
```

Comment:
Is this step necessary? I managed to get it up and running just by uploading stw.ttl

Reply:
Oh, I think I had been doing that one out of costume, without confirming whether that was necessary or not :smile: removed.